polaris.dme

```diff
@@ -1420,8 +1420,8 @@
 #include "code\modules\ai\ai_holder_targeting.dm"
 #include "code\modules\ai\interfaces.dm"
 #include "code\modules\ai\say_list.dm"
-#include "code\modules\ai\ai_holder_subtypes\simple_mob_ai.dm"
-#include "code\modules\ai\ai_holder_subtypes\slime_xenobio_ai.dm"
+#include "code\modules\ai\aI_holder_subtypes\simple_mob_ai.dm"

```

Comment:
This won't explodify folders right?

Reply:
Apparently the folders are already named with a capital I?
![evidence](https://puu.sh/FQvNb/b783696907.png)